floyd/cli/run.py

```diff
@@ -222,3 +238,74 @@ def get_command_line(instance_type, env, message, data, mode, open, tensorboard,
         if command_str:
             floyd_command.append(shell_quote(command_str))
     return ' '.join(floyd_command)
+
+
+@click.command()
+@click.argument('job_name', nargs=1)
+@click.option('--data', multiple=True, help='Data source id to use')
```

Comment:
Specifying this parameter overrides the data mounts of the original job, right? The help text should reflect that.